lib/elixir/pages/Deprecations.md

```diff
@@ -15,6 +15,7 @@ Elixir deprecations happen in 3 steps:
 
 Deprecated feature                               | Deprecated in | Replaced by (available since)
 :----------------------------------------------- | :------------ | :----------------------------
+`%{key => value}` in typespecs                   | [v1.6]        | `%{required(key) => value}` or `%{optional(key) => value}` (OTP 19)
```

Comment:
I think we need to list the first Elixir version that allows such syntax. I believe it is v1.4 (and we need to say it requires OTP 19 as well).